src/Data/Lens/Lens/Unit.purs

```diff
@@ -5,5 +5,11 @@ import Prelude
 import Data.Lens.Lens (Lens', lens)
 
 -- | There is a `Unit` in everything.
+-- | ```purescript
+-- | > view united [1,2,3]
+-- | Unit
```

Comment:
We should lower-case this as it's a value, not a type
```suggestion
-- | unit
```

Reply:
I don't remember if I did something strange here, but I think i pasted from the repl what I got, are you sure it shouldn't be the constructor `Unit`?